lib/base.js

```diff
@@ -78,14 +78,11 @@ class BaseModel {
         this[dirty].forEach((key) => {
             data[key] = this[rowData][key];
         });
-        delete data.id;
+        data.id = this.id;
 
         const datastoreConfig = {
             table: this[table],
-            params: {
-                id: this.id,
-                data
-            }
+            params: data
```

Comment:
What's the expected behavior if nothing needs to be updated? Right now it will fail validation since there is only 1 key in `params`: https://github.com/screwdriver-cd/data-schema/blob/a5e3894079a6284cc441e72f9f6918e991a0e423/plugins/datastore.js#L16